gnosis-imap/gnosis-imap-spec.ini

```diff
@@ -72,7 +72,137 @@ localMem: M
     andBool SIGS_BUF ==K selectRange(M, SIGS_LOC +Int 32, SIGS_LEN)
 +attribute: [trusted, matching(#gas)]
 
-[encodeTransactionData]
+[encodeTransactionData-internal]
+; output = bytes32(32) bytes32(66) bytes1(0x19) bytes1(0x1) bytes32(DOMAIN_SEPARATOR) bytes32(SAFE_TX_HASH) bytes30(0)
+; size = 160
+k: #execute ~> _
+output: _
+statusCode: _
+callStack: _
+this: #PROXY_ID
+msg_sender: MSG_SENDER
+callData: _
+callValue: 0
+wordStack: NONCE : REFUND_RECEIVER : GAS_TOKEN : GAS_PRICE : DATA_GAS : SAFE_TX_GAS : OPERATION
+                 : DATA_LEN_LOC : VALUE : TO : RETURN_LOC : WS
+           => RETURN_LOC : INIT_USED_MEM_PTR +Int 384 : WS
+localMem:INIT_MEM =>
+  storeRange( storeRange( storeRange( storeRange( storeRange( storeRange( storeRange(
+  storeRange( storeRange( storeRange( storeRange( storeRange( storeRange( storeRange(
+  storeRange( storeRange( storeRange( storeRange(
+  storeRange( INIT_MEM,
+    ; internal computations, memory no longer used
+    INIT_USED_MEM_PTR +Int 32,  32, _),
+    INIT_USED_MEM_PTR +Int 64,  32, _),
+    INIT_USED_MEM_PTR +Int 96,  32, _),
+    INIT_USED_MEM_PTR +Int 128, 32, _),
+    INIT_USED_MEM_PTR +Int 160, 32, _),
+    INIT_USED_MEM_PTR +Int 192, 32, _),
+    INIT_USED_MEM_PTR +Int 224, 32, _),
+    INIT_USED_MEM_PTR +Int 256, 32, _),
+    INIT_USED_MEM_PTR +Int 288, 32, _),
+    INIT_USED_MEM_PTR +Int 320, 32, _),
+    INIT_USED_MEM_PTR +Int 352, 32, _),
+    INIT_USED_MEM_PTR,          32, _),
+    64,                     32, _),
+
+    ;start of result, divided into 4 parts:
+    ;return abi.encodePacked(byte(0x19), byte(0x01), domainSeparator, safeTxHash)
+    INIT_USED_MEM_PTR +Int 416, 32, #padRightToWidth(32, #parseByteStack("0x19")) ),
+    INIT_USED_MEM_PTR +Int 417, 32, #padRightToWidth(32, #parseByteStack("0x01")) ),
+    INIT_USED_MEM_PTR +Int 418, 32, #buf(32, DOMAIN_SEPARATOR) ),
+    INIT_USED_MEM_PTR +Int 450, 32, #buf(32, SAFE_TX_HASH ) ),
+    ;output len
+    INIT_USED_MEM_PTR +Int 384, 32, #padToWidth(32, #asByteStack(66)) ),
```

Comment:
why not `#buf(32, 66)`?

we have:
```
rule #buf(32, DATA) => #padToWidth(32, #asByteStack(DATA))  [concrete]
```
at https://github.com/runtimeverification/verified-smart-contracts/blob/master/gnosis/verification.k#L195